pymessenger/bot.py

```diff
@@ -5,14 +5,18 @@
 DEFAULT_API_VERSION = 2.6
 
 class Bot(object):
-    def __init__(self, access_token, api_version=DEFAULT_API_VERSION):
+    def __init__(self, access_token, api_version=DEFAULT_API_VERSION, app_secret=None):
         self.api_version = api_version
         self.access_token = access_token
         self.base_url = (
             "https://graph.facebook.com"
             "/v{0}/me/messages?access_token={1}"
         ).format(self.api_version, access_token)
 
+        if app_secret is not None:
```

Comment:
I like this! This is definitely something we should have added into the wrapper at the start! :+1: